City-of-Helsinki/employment-services-ui
# THF-484: Update HDS

Updated HDS from 2.10 to 2.13. This version improved CSS quality.

How to tests:

- Go through the pages and see that they match the production
- Create example basic page with all components and see that everything works as planned.
